AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m

```diff
@@ -355,6 +355,9 @@ - (void)videoTileDidAddWithTileState:(VideoTileState *)tileState {
 - (void)videoTileDidRemoveWithTileState:(VideoTileState *)tileState {
     [self.logger infoWithMsg:[NSString stringWithFormat:@"Removing Video Tile tileId: %ld, attendeeId: %@", (long)tileState.tileId, tileState.attendeeId]];
     [self.meetingSession.audioVideo unbindVideoViewWithTileId:tileState.tileId];
+    if (![tileState isLocalTile]) {
+        [self.remoteVideoView resetImage];
```

Comment:
Why only for remote attendee?

Reply:
Because we haven't implement the feature to stopLocalVideo in ObjC, so once it's added it will never be removed.